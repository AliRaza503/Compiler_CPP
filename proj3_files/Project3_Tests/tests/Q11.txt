{
    if(+ * x y < a b){
        a = b;
    }
    switch(+ 1 1){
        case 1: {
            x = true;
        }
    }
    a = + 1 1;
    b = + true true;
}