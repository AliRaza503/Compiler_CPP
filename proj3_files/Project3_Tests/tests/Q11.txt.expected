TYPE MISMATCH 11 C1
